cmd/analyzer/parse_args.go

```diff
@@ -212,8 +212,8 @@ func errorInErgs(args *InArgs, flagset *flag.FlagSet) error {
 
 func notSupportedYetArgs(args *InArgs) error {
 	diffAnalysis := *args.AnalysisType == allEndpointsDiff || *args.AnalysisType == allSubnetsDiff
-	if !diffAnalysis && *args.AnalysisType != allEndpoints && *args.OutputFormat != TEXTFormat &&
-		*args.OutputFormat != JSONFormat {
+	if !diffAnalysis && *args.AnalysisType != allEndpoints && *args.AnalysisType != allSubnets &&
```

Comment:
after you merge from main this change is not needed, instead you should update `supportedAnalysisTypesMap`.